### PR TITLE
JSONSpanSerializer add ToEscaped while serializing serviceName, span.Name, binaryAnnotation.Value

### PR DESCRIPTION
**Pull request by anthonywanted**
JSONSpanSerializer add ToEscaped while serializing serviceName, span.Name, binaryAnnotation.Value
Fix bug https://github.com/openzipkin/zipkin4net/issues/156

### CODE REVIEW

File: Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ private void SerializeSpan(StreamWriter writer, Span span)
             writer.Write(openingBrace);
             writer.WriteField(id, NumberUtils.EncodeLongToLowerHexString(span.SpanState.SpanId));
             writer.Write(comma);
-            writer.WriteField(name, span.Name ?? SerializerUtils.DefaultRpcMethodName);
+            writer.WriteField(name, span.Name == null ? SerializerUtils.ToEscaped(span.Name) : SerializerUtils.DefaultRpcMethodName);
```
Comment (fedj) [inline Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs]: I think you meant the opposite. If the name is *not* null then escape it

File: Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs [hunk 2]
```diff
@@ -126,7 +126,7 @@ private static void SerializeEndPoint(StreamWriter writer, IPEndPoint endPoint,
             writer.Write(comma);
             writer.WriteField(port, (short)endPoint.Port);
             writer.Write(comma);
-            writer.WriteField(JSONSpanSerializer.serviceName, serviceName);
+            writer.WriteField(JSONSpanSerializer.serviceName, SerializerUtils.ToEscaped(serviceName));
```
Comment (fedj) [inline Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs:129]: Shouldn't we also escape annotation.value on line 115 ?

**Review by fedj — COMMENTED** [review]
Thanks a lot for the PR. I only have 1 thing to change and a question

File: Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs (hunk 1 above)
Comment (anthonywanted) [reply Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs]: my bad

File: Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs (hunk 2 above)
Comment (anthonywanted) [reply Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs:129]: I find the same problem in serviceName too, so I add it too
I thought annotation.Value is fully controlled by us, so I didn't add it
I commit again

**Comment by fedj** [comment]
Thanks a lot @anthonywanted !

**Pull request merged**